plugin-templates/config/post-meta.json

```diff
@@ -0,0 +1,6 @@
+{
+  "create_wordpress_plugin_featured_image_caption": {
+    "post_types": "all",
```

Comment:
I wonder if there's a way to only register this for post types that support a featured image 🤔

Reply:
great idea. I added this.